<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Power</class>
 <widget class="QWidget" name="Power">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>152</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>119</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Power</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vicom_v4-7_helixed.qrc">
    <normaloff>:/images/dc2.png</normaloff>:/images/dc2.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="centerLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>34</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdAutoCtr">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>15</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnLock">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>14</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 10px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="vicom_v4-7_helixed.qrc">
         <normaloff>:/images/unlocked.png</normaloff>
         <normalon>:/images/locked.png</normalon>:/images/unlocked.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QDockWidget" name="dockWidgetPos">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"> QDockWidget {
     border: 1px solid black;
     titlebar-close-icon: url(close.png);
     titlebar-normal-icon: url(undock.png);
 }

 QDockWidget::title {
     text-align: center; /* align the text to the left */
     background: red;
     padding-left: 5px;
 }

 QDockWidget::close-button, QDockWidget::float-button {
     border: 1px solid transparent;
     background: darkgray;
     padding: 0px;
 }

 QDockWidget::close-button:hover, QDockWidget::float-button:hover {
     background: gray;
 }

 QDockWidget::close-button:pressed, QDockWidget::float-button:pressed {
     padding: 1px -1px -1px 1px;
 }
</string>
     </property>
     <property name="windowTitle">
      <string>Positive supply</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="HLed" name="ledDvmRd_2" native="true">
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-color: rgb(58, 58, 58);
background-color: rgb(195, 195, 195);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HLed" name="ledDvmGn_2" native="true">
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-color: rgb(58, 58, 58);
background-color: rgb(195, 195, 195);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLCDNumber" name="lcdVoltPos">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(167, 167, 167);

border-color: rgb(255, 0, 0);
font: 10pt &quot;Monospace&quot;;</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>5</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnEnPos">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>O</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="lcdAmpPos">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(167, 167, 167);

border-color: rgb(255, 0, 0);
font: 10pt &quot;Monospace&quot;;</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>5</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDockWidget" name="dockWidgetNeg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"> QDockWidget {
     border: 1px solid black;
     titlebar-close-icon: url(close.png);
     titlebar-normal-icon: url(undock.png);
 }


 QDockWidget::title {
     text-align: center; 
     background: rgb(21, 199, 253);
     
	 color: white;
     padding-left: 5px;
 }
/*
 QDockWidget::close-button, QDockWidget::float-button {
     border: 1px solid transparent;
     background: darkgray;
     padding: 0px;
 }

 QDockWidget::close-button:hover, QDockWidget::float-button:hover {
     background: gray;
 }

 QDockWidget::close-button:pressed, QDockWidget::float-button:pressed {
     padding: 1px -1px -1px 1px;
 }
*/</string>
     </property>
     <property name="windowTitle">
      <string>Negative supply</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="1">
          <widget class="QLCDNumber" name="lcdVoltNeg">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(167, 167, 167);

border-color: rgb(255, 0, 0);
font: 10pt &quot;Monospace&quot;;</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>5</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="lcdAmpNeg">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(167, 167, 167);

border-color: rgb(255, 0, 0);
font: 10pt &quot;Monospace&quot;;</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>5</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="HLed" name="ledDvmRd" native="true">
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-color: rgb(58, 58, 58);
background-color: rgb(195, 195, 195);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HLed" name="ledDvmGn" native="true">
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-color: rgb(58, 58, 58);
background-color: rgb(195, 195, 195);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnEnNeg">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>O</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vicom_v4-7_helixed.qrc"/>
 </resources>
 <connections/>
</ui>
